### PR TITLE
Release 1.1.0b3

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
note: the crc32 issues surfaced when building for python 3.4.0, which uses a compiler option to make that stuff an error.

File: src/borg/_crc32/clmul.c [hunk 1]
```diff
@@ -345,6 +347,13 @@ crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 
     int first = 1;
 
+    /* fold 512 to 32 */
+    const __m128i xmm_mask  = _mm_load_si128((__m128i *)crc_mask);
+    const __m128i xmm_mask2 = _mm_load_si128((__m128i *)crc_mask2);
```
Comment (enkore) [inline src/borg/_crc32/clmul.c]: Move the assignment back down below the `done:` label.

Otherwise lgmt.

File: src/borg/_crc32/clmul.c [hunk 2]
```diff
@@ -345,6 +347,13 @@ crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 
     int first = 1;
 
+    /* fold 512 to 32 */
```
Comment (enkore) [inline src/borg/_crc32/clmul.c]: `/* fold 512 to 32 (variable declarations for ISO-C90) */`

File: src/borg/_crc32/clmul.c [hunk 3]
```diff
@@ -463,13 +472,6 @@ crc32_clmul(const uint8_t *src, long len, uint32_t initial_crc)
 done:
     (void)0;
 
-    /* fold 512 to 32 */
```
Comment (enkore) [inline src/borg/_crc32/clmul.c]: `/* fold 512 to 32 */`

Comment (ThomasWaldmann) [inline src/borg/_crc32/clmul.c]: fixed.

File: src/borg/_crc32/clmul.c (hunk 2 above)
Comment (ThomasWaldmann) [inline src/borg/_crc32/clmul.c]: fixed

File: MANIFEST.in [hunk 4]
```diff
@@ -2,6 +2,7 @@ include README.rst AUTHORS LICENSE CHANGES.rst MANIFEST.in
 graft src
 recursive-exclude src *.pyc
 recursive-exclude src *.pyo
+recursive-exclude src *.so
```
Comment (ThomasWaldmann) [inline MANIFEST.in]: I only did the minimal required change here.

More work should be done in 1.0-maint, then merged into master.

**Pull request merged**